### ISSUE TITLE
sharpcaster on android

### ISSUE THREAD

**Issue by noelitc** [Tapanila/SharpCaster#215]
Hi,
Thanks fo your help before. Because sharpcaster uses  Microsoft.Extensions.DependencyInjection; ,does that mean it wont comile and run on android?
regards
Noel

**Comment by noelitc** [comment]
also is there example of just sending a text message to a custom reciever?

**Comment by noelitc** [comment]
Hi,
When I said android I meant Unity built for android.
regards
Noel

**Comment by noelitc** [comment]
Here is a sample Unity Project I made with with sharpcaster.
https://github.com/noelitc/sharpcastertest

**Comment by noelitc** [comment]
hi,
 Sorry to is the correct one
https://github.com/noelitc/sharpCasterTestUnity

**Comment by Tapanila** [comment]
I noticed that I had old packages.config files in repository and removed those.
Not sure how to use the new package references on unity but if you start using those it should just work.

If you need to keep using packages.config then you can just add this there:
<package id="Microsoft.Extensions.DependencyInjection" version="7.0.0" />

**Comment by noelitc** [comment]
Thanks. but that had no impact.  is Microsoft.Extensions.DependencyInjection core to the project. Could I implement a simpler approach for my particular need. I just want to send a text message to the chromecast?

**Comment by Tapanila** [comment]
You would need to replace the dependency injection that I use with something else. It's only used in one place so shouldn't be too hard to replace it.

**Comment by Tapanila** [comment]
Seems like if you get the Microsoft.Extensions.DependencyInjection Downloaded it works pretty well. After that it seems to require to switch from .net standard 2.1 to .net framework since dynamic isn't supported on .net standard 2.1.

I opened a pull request to your repo where I can get it compile without warnings but not sure at all does it really work

**Comment by noelitc** [comment]
yes I used unity nuget to get Microsoft.Extensions.DependencyInjection . it compiles now without warnings but when build for android it hangs on moving processing map file...

**Comment by Tapanila** [comment]
Okay. Then I have no idea :(
